addons/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/handler/AbstractNetatmoThingHandler.java

```diff
@@ -0,0 +1,138 @@
+            if (signalLevel > signalThresholds.get(level)) {
+                break;
+            }
+        }
+        return level;
+    }
+
+    protected State getNAThingProperty(String chanelId) {
+        switch (chanelId) {
+            case CHANNEL_TIMEUTC:
+                return new DateTimeType(timestampToCalendar(dashboard.getTimeUtc()));
+            case CHANNEL_TEMPERATURE:
+                return new DecimalType(dashboard.getTemperature());
+            case CHANNEL_DATE_MAX_TEMP:
+                return new DateTimeType(timestampToCalendar(dashboard.getDateMaxTemp()));
+            case CHANNEL_DATE_MIN_TEMP:
+                return new DateTimeType(timestampToCalendar(dashboard.getDateMinTemp()));
+            case CHANNEL_MAX_TEMP:
+                return new DecimalType(dashboard.getMaxTemp());
+            case CHANNEL_MIN_TEMP:
+                return new DecimalType(dashboard.getMinTemp());
+            case CHANNEL_HUMIDEX:
+                return new DecimalType(WeatherUtils.getHumidex(dashboard.getTemperature(), dashboard.getHumidity()));
+            case CHANNEL_DEWPOINT:
+                return new DecimalType(WeatherUtils.getDewPoint(dashboard.getTemperature(), dashboard.getHumidity()));
+            case CHANNEL_DEWPOINTDEP:
+                Double dewpoint = WeatherUtils.getDewPoint(dashboard.getTemperature(), dashboard.getHumidity());
+                return new DecimalType(WeatherUtils.getDewPointDep(dashboard.getTemperature(), dewpoint));
+            case CHANNEL_HEATINDEX:
+                return new DecimalType(WeatherUtils.getHeatIndex(dashboard.getTemperature(), dashboard.getHumidity()));
+            case CHANNEL_PRESSURE:
+                return new DecimalType(dashboard.getPressure());
+            case CHANNEL_ABSOLUTE_PRESSURE:
+                return new DecimalType(dashboard.getAbsolutePressure());
+            case CHANNEL_CO2:
+                return new DecimalType(dashboard.getCO2());
+            case CHANNEL_HUMIDITY:
+                return new PercentType(dashboard.getHumidity().intValue());
+            case CHANNEL_NOISE:
+                return new DecimalType(dashboard.getNoise());
+
+        }
+        return null;
+    }
+
+    protected void updateChannels() {
+        logger.debug("Updating device channels");
+
+        for (Channel channel : getThing().getChannels()) {
+            String chanelId = channel.getUID().getId();
+            State state = getNAThingProperty(chanelId);
+            if (state != null) {
+                updateState(channel.getUID(), state);
+            }
+        }
+
+    }
+
+    protected Calendar timestampToCalendar(Integer netatmoTS) {
+        Calendar calendar = Calendar.getInstance();
+        calendar.setTimeInMillis(netatmoTS * 1000L);
+        return calendar;
+    }
+
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        // Most of the modules are read-only
```

Comment:
Nonetheless, you should support the REFRESH command, which should make your binding refresh the state of the channel, if possible.